docs/examples/rag_weaviate.ipynb

```diff
@@ -0,0 +1,1092 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Ag9kcX2B_atc"
+      },
+      "source": [
+        "# Performing RAG over PDFs with Weaviate and Docling\n",
+        "## A recipe 🧑‍🍳 🐥 💚\n",
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/your-username/your-repo/blob/main/notebook.ipynb)\n",
+        "\n",
+        "By Mary Newhauser, MLE @ Weaviate\n",
```

Comment:
As commit authorship is anyway captured in Git history (which can additionally track further changes down the road) I would rather omit the name from the content itself.